kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -16,11 +16,11 @@
 package org.kie.server.api.marshalling.xstream;
 
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
 import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.converters.reflection.PureJavaReflectionProvider;
```

Comment:
I believe immutability is good :) This way, we can also fix everything at one place (in the xstream object), instead of having to add custom code to multiple classes.

I haven't found any mention on the PureJavaReflectionProvider being private API, so I assumed its fine to use it.